docusaurus.config.js

```diff
@@ -113,8 +113,10 @@ module.exports = async () => {
                     // The fallback version to the right of the pipes (||)
                     // is used in cases where the version cannot be fetched from GitHub.
                     // You can assign the latest release data to any variable.
-                    REDPANDA_VERSION_22_3: latestRedpandaReleaseVersion || '22.3.11',
-                    REDPANDA_SHA_22_3: latestRedpandaReleaseCommitHash || '9eefb907c',
+                    REDPANDA_VERSION_23_1: latestRedpandaReleaseVersion || '23.1.1',
+                    REDPANDA_SHA_23_1: latestRedpandaReleaseCommitHash || '92761c',
```

Comment:
@JakeSCahill where is this commit SHA from? I can't find it when searching 
Github: https://github.com/search?q=org%3Aredpanda-data+92761c&type=commits

Reply:
it's from the release tag: https://github.com/redpanda-data/redpanda/releases/tag/v23.1.1

Reply:
Ah, thanks, it needs the last, seventh char.
```suggestion
                    REDPANDA_VERSION_23_1: latestRedpandaReleaseVersion || '23.1.1',
                    REDPANDA_SHA_23_1: latestRedpandaReleaseCommitHash || '92761c3',
```